navbuilder/tests/requirements/110.txt

```diff
@@ -2,3 +2,4 @@
 
 django==1.10.1
 django-test-without-migrations==0.4
+django-ultracache==1.10.2
```

Comment:
Should we be introducing a dependancy here to ultracache given this is a Django only module